test/fixtures/todo/json5/deserialization/number.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = [
+  {
+    name: 'binary number starting with b',
```

Comment:
I'd have named it like "binary number declared using 0b" as it's not really starting with b, but that's left to your discretion and I'm fine either way. (same in other places)